<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>tiling_method</class>
 <widget class="QWidget" name="tiling_method">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>679</width>
    <height>656</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>645</width>
     <height>631</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="main_layout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <layout class="QVBoxLayout" name="lyt_holder_img"/>
        </item>
        <item>
         <widget class="QCheckBox" name="chk_lres">
          <property name="text">
           <string>Use low-resolution image (faster processing)</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="combo_img"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <layout class="QVBoxLayout" name="lyt_holder_tree"/>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="lyt_holder_controls"/>
    </item>
    <item>
     <widget class="QPushButton" name="btn_capture">
      <property name="text">
       <string>Perform image tiling</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
